apps/organisasjon-mottak-service/src/main/java/no/nav/registre/testnorge/organisasjonmottak/domain/Postadresse.java

```diff
@@ -0,0 +1,24 @@
+package no.nav.registre.testnorge.organisasjonmottak.domain;
+
+public class Postadresse extends Adresse {
+
+    public Postadresse(no.nav.registre.testnorge.libs.avro.organiasjon.Posstadresse posstadresse) {
```

Comment:
En S for mye i PostAdresse; `public Postadresse(no.nav.registre.testnorge.libs.avro.organiasjon.Postadresse postadresse)`

Reply:
pakkenavn der har også en liten typo, skal være no.nav.registre.testnorge.libs.avro.**organisasjon**